### ISSUE TITLE
Can't pause carousel autoplay

### ISSUE THREAD

**Issue by Flaviu989** [vitalii-andriiovskyi/ngx-owl-carousel-o#138]
I have set my carousel to **autoplay: true**, but I also want to pause autoplay when the user opens a **mat-menu** from one of the carousel slides

1. I have tried changing the options, the options do change, but the carousel still autoplays:

```
@ViewChild('owlCarousel') owlCarousel: CarouselComponent; 

carouselControl(menuOpened: boolean): void {
   menuOpened ? this.owlCarousel.options.autoplay = true : this.owlCarousel.options.autoplay = false;
}
```
2. I've also tried using **startPausing(): void** and **startPlayTE(): void**, but still the carousel autoplays

```
@ViewChild('owlCarousel') owlCarousel: CarouselComponent; 

carouselControl(menuOpened: boolean): void {
   menuOpened ?  this.owlCarousel.startPausing() : this.owlCarousel.startPlayTE();
}
```
3. Lastly I've tried using this autoplayService, which seems to work, unfortunately autoplayService is private and once I restart my project it won't compile:

```
@ViewChild('owlCarousel') owlCarousel: CarouselComponent; 

carouselControl(menuOpened: boolean): void {
   menuOpened ? this.owlCarousel.autoplayService.stop() : this.owlCarousel.autoplayService.play();
}
```
Am I missing something? Does anyone know how to pause the carousel?
PS: I forgot to mention **autoplayHoverPause: true** will not work once the **mat-menu** is opened

**Comment by Callan003** [comment]
I am facing the same issue :\

**Comment by vitalii-andriiovskyi** [comment]
Hi,

Published versions `4.1.0` (for Angular 10) and `5.1.0` (for Angular 11) with solution to your problem. 
Details are on the page [Issue: `autoplay` doesn't stay paused when user opens `mat-menu`](https://github.com/vitalii-andriiovskyi/ngx-owl-carousel-o/tree/version-for-angular-11#issue-autoplay-doesnt-stay-paused-when-user-opens-mat-menu)

**Comment by Flaviu989** [comment]
> Hi,
> 
> Published versions `4.1.0` (for Angular 10) and `5.1.0` (for Angular 11) with solution to your problem.
> Details are on the page [Issue: `autoplay` doesn't stay paused when user opens `mat-menu`](https://github.com/vitalii-andriiovskyi/ngx-owl-carousel-o/tree/version-for-angular-11#issue-autoplay-doesnt-stay-paused-when-user-opens-mat-menu)

Thank you very much!
